src/pages/Events/AccessibilityReminders.tsx

```
@@ -0,0 +1,269 @@
+    details: 'Requires PDF handouts in 16pt font or higher.',
+    status: 'Urgent',
+    createdAt: '2026-08-22',
+  },
+];
+
+export default function AccessibilityReminders() {
+  const [accommodations, setAccommodations] = useState<Accommodation[]>(INITIAL_ACCOMMODATIONS);
+  const [showModal, setShowModal] = useState(false);
+  const [newRequest, setNewRequest] = useState({ user: '', requestType: '', eventDate: '', details: '' });
+
+  const pendingCount = accommodations.filter(a => a.status !== 'Confirmed').length;
```

ESLint

[error] Replace `a·=>·a.status·!==·'Confirmed'` with `(a)·=>·a.status·!==·"Confirmed"`

(prettier/prettier)